File: src/main/java/com/matamercer/microblog/web/api/activitypub/render/PersonRender.java
package com.matamercer.microblog.web.api.activitypub.render;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class PersonRender extends ActivityRender {
    private String id;
    private final String type = "Person";
    private String preferredUsername;
    private String name;
    private String inbox;
    private String outbox;
    private String liked;
    private KeyRender publicKey;
}

File: src/main/java/com/matamercer/microblog/web/api/activitypub/render/PersonRender.kt
package com.matamercer.microblog.web.api.activitypub.render

class PersonRender : ActivityRender() {
    var id: String? = null
    var type = "Person"
    var preferredUsername: String? = null
    var name: String? = null
    var inbox: String? = null
    var outbox: String? = null
    var liked: String? = null
    var publicKey: KeyRender? = null
}